platform/platform-configuration-migration/src/main/java/org/codice/ddf/configuration/migration/SystemConfigurationMigration.java

```diff
@@ -217,7 +231,12 @@ private void copyDirectory(Path source, Path destination) throws MigrationExcept
             String message = String.format(UNREAL_PATH_WARNING, propertyName, path.toString());
             LOGGER.debug(message);
             migrationWarnings.add(new MigrationWarning(message));
+        } catch (UnsupportedOperationException e) {
```

Comment:
This is for the case if it is on a Windows machine and there is no support for symbolic links